### PR TITLE
Detect format as library function

### PR DESCRIPTION
**Pull request by jakubkrafka**
What was done here:
- added detect_format as a library function
- refactored detect-format command to use the same code as a library function
- changed logic of _print() in detect-format command (removed "decision" logic about format detection -> moved to "core" function)

### CODE REVIEW

File: ocdskit/detect.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+logger = logging.getLogger('ocdskit')
+
+
+def detect_format(path, root_path=''):
+    """
+    Returns the file format with additional info whether the items are in array and/or concatenated.
+
+    :param str path: path to tested file
+    :param str root_path: ``json path`` within the file, where the tested items will be searched for
+    :return: a tuple (detected_format, is_concatenated, is_array)
```
Comment (jpmckinney) [inline ocdskit/detect.py:14]:
```suggestion
    :returns: a tuple of the format, whether data is concatenated, and whether data is in an array
    :rtype: tuple
```

File: ocdskit/util.py [hunk 2]
```diff
@@ -183,6 +186,84 @@ def _empty_package(uri, publisher, published_date, version):
     }
 
 
+def detect(path, root_path=""):
+    """
```
Comment (jpmckinney) [inline ocdskit/util.py:190]: I edited the docstring in the wrong place - meant to do it here :)

**Review by jpmckinney — COMMENTED** [review]
Looks good! Mostly edits to docs.

File: ocdskit/detect.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import logging
```
[the remark this answers is not in the document]
Comment (jakubkrafka) [reply ocdskit/detect.py:1]: That's exactly the structure question I wasn't sure about. I checked other commands and doc, and it seemed to me, that they have it's own module each. 
But, long story short - let's remove that then.

[the remark this answers is not in the document]
Comment (jakubkrafka) [reply ocdskit/detect.py]: I've removed that module in the commit bellow.

**Pull request merged**